Week2/week2_homework/grade_calculator.js

```diff
@@ -0,0 +1,39 @@
+"use strcit";
+
+
+// function to insert the score and calculate the grade & the percentage requested.
+function calculate_test_score(score) {
+  // 1- convert the score into a percentage
+  const full_score = 100;
+  const score_percentage = (score / full_score) * 100;
+
+  // 2- calculate what grade corresponds with that percentage, and
+  let grade;
+
+  // 3- shows in the command line the result: the grade and the percentage
+  switch (true) {
```

Comment:
Although this works, it is far more common to see `if..else` used in the real world for multi-expression conditional statements.